bundles/com.espressif.idf.core/src/com/espressif/idf/core/util/IDFUtil.java

```diff
@@ -43,6 +45,11 @@
  */
 public class IDFUtil
 {
+
+	private IDFUtil()
+	{
+	}
```

Comment:
Thanks for adding this. It makes sense as all the methods are static we can avoid instantiation.